docker/airflow/dags/colombia_dag.py

```diff
@@ -0,0 +1,45 @@
+from airflow.decorators import dag, task
+from airflow.operators.empty import EmptyOperator
+import pendulum
+from datetime import timedelta
+
+DATA_PATH = "/opt/EpiGraphHub/data_collection"
+import sys
+
+sys.path.insert(0, DATA_PATH)
+from colombia import load_chunks_into_db
+from config import COLOMBIA_SOC
+
+
+default_args = {
+    "owner": "epigraphhub",
+    "depends_on_past": False,
+    "start_date": pendulum.datetime(2022, 8, 26),
+    #'email': [''],
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=1),
+}
+
+
+@dag(
+    schedule_interval="@daily",
+    default_args=default_args,
+    catchup=False,
+    template_searchpath=DATA_PATH,
+)
+def colombia():
+
+    start = EmptyOperator(
+        task_id="start",
+    )
+
+    @task(task_id="load_into_db", retries=2)
```

Comment:
In this link: https://dev.socrata.com/foundry/www.datos.gov.co/gt2j-8ykr/ , it's available the last updated date of this dataset. Unfortunately, I don't find a way to get it directly from the API. But, maybe we can create a web scraping to get it and use it as a way to decide if the code to update data should run or not. I can do it.

Reply:
@luabida, a final comment: Following issue #129, you could create a task that, after updating the table, will add (or update) the date of update of this table in the meta table.